Assets/MRTK/Providers/LeapMotion/Definitions/LeapControllerOrientation.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+namespace Microsoft.MixedReality.Toolkit.LeapMotion.Input
+{
+    /// <summary>
+    /// The location of the leap motion controller. LeapControllerOrientation.Headset indicates the controller is mounted on a headset. 
```

Comment:
Great comments, though there's a bit of duplication here (i.e. it would be better to have the details of what the values are on the values themselves (as they kinda are already).).

Reply:
If you wanted to mention the ramifications of the choice though (i.e. these have the effect of determining the offset of the hand information relative to the camera), that could be interesting to note here.

Reply:
Updated